### PR TITLE
Change GitHub workflow to use latest YugabyteDB docker images

### PR DESCRIPTION
**Pull request by rahulb-yb**

### CODE REVIEW

File: .github/workflows/mysql-migtests.yml [hunk 1]
```diff
@@ -7,8 +7,10 @@ on:
     branches: [ main, v1.0.0-xbeta ]
 
 jobs:
-
   run-mysql-migration-tests:
+    strategy:
+      matrix:
+        version: [2.14.2.0-b25, 2.15.2.0-b87]
```
Comment (sanyamsinghal) [inline .github/workflows/mysql-migtests.yml:13]: Is it possible to do something like: For odd versions can we fetch the latest yb version available and use that.

**Review by sanyamsinghal — APPROVED** [review]
LGTM
Left a minor comment.

File: .github/workflows/mysql-migtests.yml (hunk 1 above)
Comment (rahulb-yb) [reply .github/workflows/mysql-migtests.yml:13]: we can use the "latest" tag, but there is no guarantee of it being the latest odd version, so decided to explicitly use latest two versions as of now

Comment (sanyamsinghal) [reply .github/workflows/mysql-migtests.yml:13]: ok

**Pull request merged**